Nodejs/Product/Nodejs/NodejsPackage.cs

```diff
@@ -94,11 +94,16 @@ protected override void Initialize()
             RegisterProjectFactory(new NodejsProjectFactory(this));
 
             // Add our command handlers for menu (commands must exist in the .vsct file)
-            var commands = new List<Command> {
-                new ImportWizardCommand(),
-            };            
+            var importWizardCmd = new List<Command> {
```

Comment:
This is not wrong, but I would prefer migration to be in its own package, so we can easily remove it when it's no longer needed (migration should be a temporary thing, I believe)

Reply:
i'm still working on this -- will update this comment once that is done

Reply:
Resolved offline -- decided against moving the migration into its own package, but will refactor to keep migration logic as easily separable as possible from the rest of nodejstools (i.e. move migrate command to its own .vsct file).

Reply:
Moved to item #&#x2060;1 in #2456